src/com/homework/lesson10/Main.java

```diff
@@ -0,0 +1,76 @@
+package com.homework.lesson10;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.http.HttpClient;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class Main {
+    static long startCount2=0;
+    public static void main(String[] args) throws InterruptedException {
+        List<String> https = new CopyOnWriteArrayList<>();
+        https.add("https://habr.com/");
+        https.add("https://github.com/");
+        https.add("https://www.google.com/");
+        long start = System.currentTimeMillis();
+        long startCount = 0;
+        for (String http : https) {
+            try (InputStream inputStream = new URL(http).openStream()) {
+                final byte[] bytes = inputStream.readAllBytes();
+                startCount += bytes.length;
+                System.out.println(startCount);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+
+        }
+        long finish = System.currentTimeMillis();
+        long elapsed = finish - start;
+        System.out.println("Прошло времени, мс: " + elapsed);
+        System.out.println("Общий размер: " + startCount);
+
+        long start2 = System.currentTimeMillis();
+        Thread thread1 = new Thread(() -> {
+            try (InputStream inputStream = new URL(https.get(0)).openStream()) {
+                final byte[] bytes = inputStream.readAllBytes();
+                startCount2 += bytes.length;
```

Comment:
Based on the comment above, here you need to push bytes to the collection (or bytes.length, its not important for this task)